include/picongpu/simulation_defines.hpp

```diff
@@ -53,6 +53,10 @@ namespace picongpu
 // ##### load unitless
 #include <picongpu/_defaultUnitless.loader>
 #include <picongpu/extensionUnitless.loader>
+
+// is added last, so that any possible metadata from the param files can be added:
+#include "picongpu/param/metadata.param"
```

Comment:
typically we include params in `_defaultParam.loader` except for a few params where we need to solve cyclic dependencies.

Reply:
Offline discussion: It is yet unclear if there is a need to have this configurable. As long as we don't know, we will spare us the maintenance overhead and make this a regular header.